website/versioned_docs/version-26.x/MockFunctions.md

```diff
@@ -75,6 +83,9 @@ expect(someMockFunction.mock.instances.length).toBe(2);
 // The object returned by the first instantiation of this function
 // had a `name` property whose value was set to 'test'
 expect(someMockFunction.mock.instances[0].name).toBe('test');
+
+// The first argument of the last call to the function was 'test'
+expect(someMockFunction.mock.lastCall[0]).toBe('test');
```

Comment:
Please take a look at [Files Changed tab](https://github.com/facebook/jest/pull/13428/files). Some accidental changes are still left (like these lines).

Reply:
Good idea I deleted accidental changes to the V25. please review again if correct I'll do the same to the other versions.